setup/gcp/installer/util/gcloud.py

```diff
@@ -608,9 +609,29 @@ def get_forseti_server_info():
     ip_addr, zone, name = get_vm_instance_info('forseti-security-server',
                                                try_match=True)
 
-    return ('', 'us-central1-c', '') if ip_addr is None else (ip_addr,
-                                                              zone,
-                                                              name)
+    if ip_addr is None:
+        print('No forseti server detected, you will need to install'
+              ' forseti server before installing the client, exiting...')
+        sys.exit(1)
+
+    return ip_addr, zone, name
+
+
+def get_forseti_v1_info():
```

Comment:
I don't think that you need a separate method here.  Why not just add the regex string to inside of `get_vm_instance_info()`?

Reply:
Updated.